data/human/variants.txt

```diff
@@ -2352,6 +2374,22 @@ ship "Headhunter" "Headhunter (Strike)"
 		"Hyperdrive"
 
 
+ship "Headhunter" "Headhunter (Tractor Beam)"
```

Comment:
Headhunters are quick enough to be able to chase flotsams without needing to spend precious LW outfit space on a tractor beam. Maybe it should be added to the Vanguard instead, like you mentioned in the PR description?

Reply:
Fair enough. It's good that they also have a cargo expansion, then. :)